libs/sdk-core/src/breez_services.rs

```diff
@@ -451,9 +451,10 @@ impl BreezServices {
         filter: PaymentTypeFilter,
         from_timestamp: Option<i64>,
         to_timestamp: Option<i64>,
+        amount_of_payments: Option<u32>,
```

Comment:
Perhaps we can pass here two parameters? offset, limit
This will make it more generic and add full paging support.

Reply:
I added the field in [this commit](https://github.com/breez/breez-sdk/compare/4abb0b97f4e35971e4a0aa82c9711051ab3ceb53..1e3f7c6ef0b278a21206b1fab9d075525713cdf9) but then the parameter list got very long, so I decided to restructure it [here](https://github.com/breez/breez-sdk/pull/460/commits/6254b0a62b17fc7598cf13973da15f021ff9be33).

Reply:
This is the right decision! Please be aware that this was also changed in main branch to a struct.